scripts/globals/spells/absorb-acc.lua

```diff
@@ -26,8 +26,8 @@ function onSpellCast(caster,target,spell)
 			spell:setMsg(85);
 		else
 			spell:setMsg(533);
-			caster:addStatusEffect(EFFECT_ACCURACY_BOOST,ABSORB_SPELL_AMOUNT*resist, ABSORB_SPELL_TICK, ABSORB_SPELL_AMOUNT*ABSORB_SPELL_TICK,FLAG_DISPELABLE); -- caster gains ACC
-			target:addStatusEffect(EFFECT_ACCURACY_DOWN,ABSORB_SPELL_AMOUNT*resist, ABSORB_SPELL_TICK, ABSORB_SPELL_AMOUNT*ABSORB_SPELL_TICK,FLAG_ERASBLE);    -- target loses ACC
+			caster:addStatusEffect(EFFECT_ACCURACY_BOOST,ABSORB_SPELL_AMOUNT*resist/100*(100+(caster:getMod(MOD_LIBERATOR_ABSORB_BONUS))), ABSORB_SPELL_TICK, ABSORB_SPELL_AMOUNT*ABSORB_SPELL_TICK,FLAG_DISPELABLE); -- caster gains ACC
+			target:addStatusEffect(EFFECT_ACCURACY_DOWN,ABSORB_SPELL_AMOUNT*resist/100*(100+(caster:getMod(MOD_LIBERATOR_ABSORB_BONUS))), ABSORB_SPELL_TICK, ABSORB_SPELL_AMOUNT*ABSORB_SPELL_TICK,FLAG_ERASBLE);    -- target loses ACC

```

Comment:
this is pretty hard to read.. I'm assuming the intention is to add a percent bonus of MOD_LIBERATOR_ABSORB_BONUS, but the way the division and multiplication is set up looks strange
i'd rather see ABSORB_SPELL_AMOUNT*resist*((100+(caster:getMod(MOD_LIBERATOR_ABSORB_BONUS)))/100)

while we're at it, don't call it LIBERATOR_BONUS - the point of the mod is to be able to decouple it from the specific weapon if needed